sources/packages/web/src/components/aest/students/modals/UpdateDisabilityStatusModal.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <v-form ref="updateDisabilityStatusForm">
+    <modal-dialog-base
+      title="Update disability status"
+      :showDialog="showDialog"
+    >
+      <template #content>
+        <error-summary :errors="updateDisabilityStatusForm.errors" />
+        <p class="label-value">
+          Update the disability status of student and add note.
+        </p>
+        <v-select
```

Comment:
We missed many places but I believe that the idea was to use the `hide-details="auto"` to save some vertical spacing. Can we add it? It is the same used for the `v-textarea`.